[Docs] Revise Vizro-MCP docs for Cursor (#1354)
mckinsey/vizro: vizro-mcp/docs/pages/guides/use-prompt-templates.md

!!! warning "Check if the templates are available for your MCP host"

    Vizro-MCP's prompt templates are available in Claude Desktop and Cursor, but may not allowed by other MCP hosts.

You can use the prompt templates to create specific dashboards quickly.

In Claude Desktop, click on the plus icon below the chat, and choose _`Add from vizro-mcp`_.

![Claude Desktop MCP Server Icon](../../assets/images/claude_prompt.png)

In Cursor, access the prompt templates by typing `/` into the prompt, which will bring up a list of prompts available:

![](../../assets/images/use-cursor-prompt-template.png)

> The templates available currently as as follows:

Reviewer: ```suggestion
The templates available currently are as follows:
```

> - **`create_dashboard`** Creates a dashboard based on a local or remote CSV dataset.

Reviewer: It's not limited to CSV right? It could also take an excel file, as long as it can be converted to a pandas dataframe

```suggestion
- **`create_dashboard`** Creates a dashboard based on a local or remote dataset.
```

> - **`create_starter_dashboard`** Get started with Vizro dashboards.

Reviewer: Correct me if I am wrong, but this creates a specific example dashboard right based on the XXX dataset? Or is this a general dashboard starter that can use any data set?

```suggestion
- **`create_starter_dashboard`** Get started with an example Vizro dashboard.
```

> - **`create_vizro_chart`** Creates a Vizro styled Plotly chart based on a local or remote CSV dataset.

Reviewer: ```suggestion
- **`create_vizro_chart`** Creates a Vizro styled Plotly chart based on a local or remote dataset.
```
